packages/for-ui/src/table/Table.tsx

```diff
@@ -21,6 +21,7 @@ import { Checkbox } from '../checkbox';
 import { Radio } from '../radio';
 import { TableCell } from './TableCell';
 import { TablePagination } from './TablePagination';
+import { LegacyText } from '../typography';
```

Comment:
```suggestion
import { Text } from '../text';
```

LegacyTextではなくTextのほうを使ってほしいです！